src/test/java/com/jcabi/github/mock/MkReleaseAssetTest.java

```diff
@@ -181,27 +178,24 @@ public void canPatchJson() throws Exception {
     }
 
     /**
-     * MkReleaseAsset should be able to fetch its raw representation.
+     * Should return the Base64-encoded value of the input contents. When
+     * decoded, should be equal to the input.
      *
-     * @throws Exception if some problem inside
+     * @throws Exception Unexpected.
      */
     @Test
-    public void fetchesRawRepresentation() throws Exception {
-        final String fetch = "fetch";
-        final ReleaseAssets assets = release().assets();
-        final ReleaseAsset asset = assets.upload(
-            fetch.getBytes(), "text/plain", "raw.txt"
-        );
-        final InputStream raw = new ByteArrayInputStream(
-            DatatypeConverter.parseBase64Binary(
-                fetch
-            )
-        );
+    public void canDecodeContentsIntoOriginalValue() throws Exception {
```

Comment:
@llorllale I think you should keep the original name "fetchesRawRepresentation", since that's that it does; You are the one decoding it, in order to assert it

Reply:
@amihaiemil ok